main.go

```diff
@@ -75,14 +75,20 @@ func generate(name string, args []string) {
 		log.Fatalln("error parsing version:", err)
 	}
 
-	addExtRegexp, err := regexp.Compile(*addext)
-	if err != nil {
-		log.Fatalln("error parsing extension inclusion regexp:", err)
+	var addExtRegexp *regexp.Regexp = nil
+	if *addext != "" {
```

Comment:
In #83, you updated README.md only. What about the actual flag descriptions in main.go? Did you forget about them, or leave them on purpose?